pytorch_lightning/trainer/connectors/logger_connector/logger_connector.py

```diff
@@ -185,9 +183,6 @@ def cache_training_step_metrics(self, opt_closure_result):
             batch_log_metrics = opt_closure_result.training_step_output.log_metrics
             logged_metrics_tmp.update(batch_log_metrics)
 
-            callback_metrics = opt_closure_result.training_step_output.callback_metrics
```

Comment:
Why can we remove this ?

Reply:
Because `training_step_output.callback_metrics` was removed (is now empty)